README.md

```diff
@@ -42,70 +42,29 @@ If you’re looking to contribute, have questions, or want to keep up-to-date ab
 
 Make sure to have the latest version of JDK 1.8 installed.
 
-Add it in your root `build.gradle` at the end of repositories.
+Add this in your root `build.gradle` at the end of repositories.
 
 ```groovy
 allprojects {
     repositories {
+        mavenCentral()
         jcenter()
+        maven { url "https://dl.bintray.com/arrow-kt/arrow-kt/" } // for SNAPSHOT builds
```

Comment:
I'd say the comment is in the wrong place. The repository that carries snapshots is the one below.